README.md

```diff
@@ -68,24 +68,24 @@ in Sass):
 #### `asset-path($relative-asset-path)`
 Returns a string to the asset.
 
-* `asset-path("rails.png")` becomes `"/assets/rails.png"`
```

Comment:
Maybe something like `is output as ` ? I suggested the change because I mis-understood "becomes" when I converted one of my rails projects to use the asset pipeline.